ml-proto/test/block.wast

```diff
@@ -46,43 +46,62 @@
     (f32.gt (block (call $nop) (f32.const 3)) (block (call $nop) (f32.const 3)))
   )
 
-  (func "br-bare" (result i32)
+  (func "break-bare" (result i32)
```

Comment:
Please do not rename "br" to "break" in the tests; the instruction is named "br".